### PR TITLE
Add tests, lint and security in github actions.

### PR DESCRIPTION
**Pull request by ilyabyar**
Add tests, lint and security in github actions

* Add `rubocop-rails` to checks(it wasn't working before)
* Add `rubocop-performance` to Rubocop. 
* Make ruby version in Gemfile, matching `.ruby-version`
* Temp disable of some Rubocop offenses

* Add Github actions:
  * Security checks
  * Rubocop
  * Eager loading
  * Tests

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '3.0.2'
+ruby '3.0.0'
```
Comment (WhatWasThatt) [inline Gemfile:4]: is it make sense to change the ruby version?

Comment (ilyabyar) [reply Gemfile:4]: When I tried to install gems for a project it said me that we have a mismatch in `.ruby-version` and `Gemfile`'s Ruby version

So, which one is the correct one?

Comment (WhatWasThatt) [reply Gemfile:4]: At the moment we are using version 3.0.2, but it's not a problem to upgrade to another version

Comment (ilyabyar) [reply Gemfile:4]: If 3.0.2 is the version that you use, I'll make it another way: 
I'll change `.ruby-version` to 3.0.2

Comment (WhatWasThatt) [reply Gemfile]: Thanks a lot

**Review by WhatWasThatt — APPROVED** [review]

**Review by dmitrymalllpr — APPROVED** [review]

**Review by twill11 — APPROVED** [review]

**Pull request merged**